# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/RetainedDisplayListHelpers.h
# Commit: 900ceaf4d39c
# Full Hash: 900ceaf4d39c8968eb4ba8c7769b47064f34ad18
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-04-29 09:53:17
# Regressor Bug: 1544948
# File Overlap Count: 1
# Description:
#   Bug 1544948 - Skip merging display lists that we're sure can't have changed. r=miko
#   
#   ComputeRebuildRegion sets ForceDescendIntoIfVisible on all modified frames and their ancestors,
#   so we can use this to detect if a display list might have modified children by looking for this
#   flag on the container item.
# ==============================================================================

diff -r 90c4bb8c0d5c -r 900ceaf4d39c layout/painting/RetainedDisplayListHelpers.h
--- a/layout/painting/RetainedDisplayListHelpers.h	Mon Apr 29 00:20:30 2019 +0000
+++ b/layout/painting/RetainedDisplayListHelpers.h	Mon Apr 29 03:14:49 2019 +0000
@@ -169,6 +169,7 @@
   MergedListIndex mIndex;
   bool mUsed;
   bool mDiscarded;
+  bool mOwnsItem;
 };
 
 bool AnyContentAncestorModified(nsIFrame* aFrame,